src/content/peerconnection/pc1/js/main.js

```diff
@@ -68,6 +71,7 @@ function gotStream(stream) {
 
 function start() {
   trace('Requesting local stream');
+  webAudio.start();
```

Comment:
I think the start method is part of the constructor here https://github.com/hlundin/samples/pull/1/files#diff-94875d83850a6447d1dcdd704357619cR25